src/js/Tabs.js

```diff
@@ -214,25 +216,32 @@ function Tabs(rootEl, config) {
 	init();
 }
 
-Tabs.init = function(el, config) {
-	const tabs = [];
-	let tEls;
-	let c;
-	let l;
-	if (!el) {
-		el = document.body;
-	} else if (!(el instanceof HTMLElement)) {
-		el = document.querySelector(el);
-	}
-	if (el.querySelectorAll) {
-		tEls = el.querySelectorAll('[data-o-component=o-tabs]');
-		for (c = 0, l = tEls.length; c < l; c++) {
-			if (!tEls[c].matches('[data-o-tabs-autoconstruct=false]') && !tEls[c].hasAttribute('data-o-tabs--js')) {
-				tabs.push(new Tabs(tEls[c], config));
-			}
+Tabs.init = function(rootEl, config) {
+	if (!rootEl) {
+		rootEl = document.body;
+	}
+	if (!(rootEl instanceof HTMLElement)) {
+		rootEl = document.querySelector(rootEl);
+	}
+
+	if (rootEl instanceof HTMLElement && /\bo-tabs\b/.test(rootEl.getAttribute('data-o-component'))) {
+		if (!rootEl.matches('[data-o-tabs-autoconstruct=false]') && !rootEl.hasAttribute('data-o-tabs--js')) {
+			return new Tabs(rootEl, config);
 		}
 	}
-	return tabs;
+
+	if (rootEl.querySelectorAll) {
+		let tabs = [];
+		const tabEls = Array.from(rootEl.querySelectorAll('[data-o-component=o-tabs]'));
+
+		tabEls.forEach(tabEl => {
```

Comment:
`Array.from` can take a second parameter, a mapping function, which it calls for each item being added to the array. Using the second parameter avoids having to iterate over the list twice.

This section could be written like this:
```js
if (rootEl.querySelectorAll) {
    const tabElements = rootEl.querySelectorAll(
        '[data-o-component=o-tabs]:not([data-o-tabs-autoconstruct=false][data-o-tabs--js])'
    );

    return Array.from(tabElements, function(tabElement) {
        return new Tabs(tabElement, config);
    });
}
```